motionplan/planManager.go

```diff
@@ -800,12 +788,35 @@ func (pm *planManager) planRelativeWaypoint(ctx context.Context, request *PlanRe
 		return nil, err
 	}
 	goalPos := tf.(*referenceframe.PoseInFrame).Pose()
+
+	// We need to create a separate ephemeral framesystem to place the
```

Comment:
I don't think this is the right place for this. This should probably be in the move request, not the `motionplan` package.

Reply:
> This should probably be in the move request, not the motionplan package.

It is not clear to me how this would work by store the desired frame-system within the move-request.
The frame-system which is passed in here may be of the following form:
```
FS := world -- PTGframe -- camera -- ...
```

------------------------------------

I imagine by your comment you mean why not pass in a frame-system of the following form:
```
FS := world -- staticTransformFrame -- PTGframe -- camera -- ...
```
This would not work because we would then be composing PTG inputs against the pose stored in the staticTransform since it is upstream.

I argue that this is actually the right place for this code to belong.
My reasoning is that we are in the `planRelativeWaypoint` code path, so we know for sure that we are dealing with PTGs.

Reply:
I do mean passing in:
> `FS := world -- staticTransformFrame -- PTGframe -- camera -- ...`
The composition against the static transform can be dealt with, because it'll be known. We enforce that there can't be nested relative frames, so it would be possible to e.g. re-root the frame system on the relative frame, or deal with this specially in the solver frame, etc.

What I don't like here is the construction of a different FS for a one-off operation of setting up the planner, where we're changing the state of the planager's solverFrame and editing that back and forth. That's not good.